### PR TITLE
Use become instead of sudo

### PR DESCRIPTION
**Pull request by dasJ**

### CODE REVIEW

**Review by SlothOfAnarchy — APPROVED** [review]

File: tasks/olc-setup.yml [hunk 1]
```diff
@@ -37,7 +37,9 @@
 
 # Not using become the user may be overridden
 - name: Apply basic OLC configuration
-  command: "/usr/bin/sudo -u {{ slapd_user }} /usr/sbin/slaptest -f {{ olc_tmp }} -F {{ slapd_olc_dir }}"
+  command: "/usr/sbin/slaptest -f {{ olc_tmp }} -F {{ slapd_olc_dir }}"
+  become: yes
+  becom_user: "{{ slapd_user }}"
```
Comment (SlothOfAnarchy) [inline tasks/olc-setup.yml]: become_user

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: tasks/olc-setup.yml (hunk 1 above)
Comment (dasJ) [reply tasks/olc-setup.yml]: Whoops #untested

**Comment by dasJ** [comment]
@SlothOfAnarchy Pls rereview

**Comment by SlothOfAnarchy** [comment]
I thought we didn't want to force push on top of already reviewed code?

**Review by SlothOfAnarchy — APPROVED** [review]

**Comment by dasJ** [comment]
@SlothOfAnarchy That's what **I** usually say to you

**Comment by SlothOfAnarchy** [comment]
@dasJ "do as I say, not as I do"

**Pull request merged**